lib/generators/suspenders/testing_generator.rb

```diff
@@ -0,0 +1,84 @@
+module Suspenders
+  module Generators
+    class TestingGenerator < Rails::Generators::Base
+      source_root File.expand_path("../../templates/testing", __FILE__)
+      desc "Set up the project for an in-depth test-driven development workflow."
+
+      def add_gems
+        gem_group :development, :test do
+          gem "rspec-rails", "~> 6.1.0"
+        end
+
+        gem_group :test do
+          gem "action_dispatch-testing-integration-capybara",
+            github: "thoughtbot/action_dispatch-testing-integration-capybara", tag: "v0.1.0",
+            require: "action_dispatch/testing/integration/capybara/rspec"
+          gem "shoulda-matchers", "~> 6.0"
+          gem "webdrivers"
+          gem "webmock"
+        end
+
+        Bundler.with_unbundled_env { run "bundle install" }
+      end
+
+      def run_rspec_installation_script
+        rails_command "generate rspec:install"
+      end
+
+      def modify_rails_helper
+        insert_into_file "spec/rails_helper.rb",
+          "\s\sconfig.infer_base_class_for_anonymous_controllers = false\n",
+          after: "RSpec.configure do |config|\n"
+      end
+
+      def modify_spec_helper
+        persistence_file_path = "\s\sconfig.example_status_persistence_file_path = \"tmp/rspec_examples.txt\"\n"
+        order = "\s\sconfig.order = :random\n\n"
+        webmock_config = <<~RUBY
+
+          WebMock.disable_net_connect!(
+            allow_localhost: true,
+            allow: [
+              /(chromedriver|storage).googleapis.com/,
+              "googlechromelabs.github.io",
+            ]
```

Comment:
I found I needed to add `googlechromelabs.github.io` and adjust the original host when I tested this on a fresh Rails app. Otherwise, we would raise `WebMock::NetConnectNotAllowedError`.